cmd/utils/flags.go

```diff
@@ -259,9 +267,11 @@ var (
 		Usage: "The layer of tries trees that keep in memory",
 		Value: 128,
 	}
-	AllowInsecureNoTriesFlag = cli.BoolTFlag{
-		Name:  "allow-insecure-no-tries",
-		Usage: `Disable the tries state root verification, the state consistency is no longer 100% guaranteed, diffsync is not allowed if enabled. Do not enable it unless you know exactly what the consequence it will cause.`,
+	defaultVerifyMode = ethconfig.Defaults.TriesVerifyMode
+	TriesVerifyModeFlag = TextMarshalerFlag{
+		Name: "tries-verify-mode",
+		Usage: `tries verify mode: "local", "full", "light", "insecure"`,
```

Comment:
please explain in details what the each mode means in the `usage` so that the user can understand